apps/system/js/value_selector/value_selector.js

```diff
@@ -232,7 +232,7 @@
     }
     this.element.blur();
     this.element.hidden = true;
-    if (this.app) {
+    if (this.app && this.app.isActive()) {
```

Comment:
Hmm...I just notice we should use this.app.getBottomMostWindow().isActive() && this.app.isActive()

To be more strict, even bottom most window is active does not mean it's top most, so maybe something like appTransitionController.shouldFocus....?